src/AppInstallerRepositoryCore/Microsoft/SQLiteIndex.cpp

```diff
@@ -111,7 +111,7 @@ namespace AppInstaller::Repository::Microsoft
     {
         m_dbconn.EnableICU();
         m_version = Schema::Version::GetSchemaVersion(m_dbconn);
-        AICLI_LOG(Repo, Info, << "Opened SQLite Index with version [" << m_version << "], last write [" << GetLastWriteTime() << "]");
+        AICLI_LOG(Repo, Info, << "Opened SQLite Index with version [" << m_version << "], last write [" << GetLastWriteTime().time_since_epoch().count() << "]");

```

Comment:
Again, the point of the output operators is to simplify the call sites.  This is not.